README.md

```diff
@@ -110,7 +110,8 @@ const nsB = {
     }
   }
 }
-
+// you can optionaly provide a second argument, with options:
+// { optimize: boolean } - if true (default) it will optimize the interface, to be `enableSelector: "optimize"` compatible (see TypeScript i18next docs)
```

Comment:
are you sure it is the default?

Reply:
It is not! good catch. Damn autocmplete being to eager.